.github/actions/setup/action.yml

```diff
@@ -40,6 +40,8 @@ runs:
         sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-13 100
```

Comment:
Typo in commit message: `VCPKG_FORCE_SYSTEM_BINAIRES`

Reply:
Also a typo about setting it to "2" instead of "1" 🤦 .